plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -198,13 +198,15 @@ def get_metadata(self):
         try:
             metadata_files.download_repomd()
         except IOError, e:
-            raise FailedException(str(e))
+            raise PulpCodedException(error_code=error_codes.RPM1004, reason=str(e))
```

Comment:
The platform does that for us now, once this is merged: https://github.com/pulp/pulp/pull/1772/